### PR TITLE
Code coverage/quality reporting

### PR DESCRIPTION
**Pull request by skyzyx**
* Adds code coverage reporting with [CodeCov](https://codecov.io/gh/wepay/kafka-connect-bigquery).
* Adds code quality reporting with [Codacy](https://www.codacy.com/app/skyzyx/kafka-connect-bigquery/dashboard) (although we still need to tweak the review rules).
* Adds @mtagle as an author (as per the `build.gradle` file).
* Adds link/badge for Maven Central (#17).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,6 +5,7 @@ apply plugin: 'findbugs'
 apply plugin: 'jacoco'
 apply plugin: 'maven'
 apply plugin: 'signing'
+apply plugin: 'jacoco'
```
Comment (C0urante) [inline build.gradle]: Already using the Jacoco plugin (see line 5).

Comment (skyzyx) [inline build.gradle]: Ack. Good catch.

File: .travis.yml [hunk 2]
```diff
@@ -10,9 +10,15 @@ jdk:
 matrix:
   fast_finish: true
 
+# before_script:
```
Comment (C0urante) [inline .travis.yml]: Is this left in intentionally?

File: .travis.yml [hunk 3]
```diff
@@ -10,9 +10,15 @@ jdk:
...
+
 script:
   - gradle test
 
+after_success:
+  - if [ -e ./gradlew ]; then ./gradlew jacocoTestReport;else gradle jacocoTestReport;fi
```
Comment (C0urante) [inline .travis.yml]: Nit: improve readability by adding spaces after semicolons ("jacocoTestReport;else" -> "jacocoTestReport; else", "jacocoTestReport;fi" -> "jacocoTestReport; fi").

File: README.md [hunk 4]
```diff
@@ -1,12 +1,16 @@
 # Kafka Connect BigQuery Connector
 
 [![Source](https://img.shields.io/badge/source-wepay/kafka–connect–bigquery-blue.svg?style=flat-square)](https://github.com/wepay/kafka-connect-bigquery)
-[![Release](https://img.shields.io/github/release/wepay/kafka-connect-bigquery.svg?style=flat-square)](https://travis-ci.org/wepay/kafka-connect-bigquery)
+[![Release](https://img.shields.io/github/release/wepay/kafka-connect-bigquery.svg?style=flat-square)](https://github.com/wepay/kafka-connect-bigquery/releases)
+[![Maven Central](https://img.shields.io/maven-central/v/com.wepay/kafka-connect-bigquery.svg?style=flat-square)](https://repo1.maven.org/maven2/com/wepay/kafka-connect-bigquery/)
```
Comment (C0urante) [inline README.md]: Sorry to change this on you, but would you mind switching the link to http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.wepay%22%20AND%20a%3A%22kafka-connect-bigquery%22 ? I noticed Debezium was doing something similar and think it probably makes more sense to lead to the Maven Central version page for the connector than the actual Maven Central repo.

File: .travis.yml (hunk 3 above)
Comment (skyzyx) [inline .travis.yml]: Good call. I pulled this from a wiki page and forgot to reformat it.

File: README.md (hunk 4 above)
Comment (skyzyx) [inline README.md]: Yeah, that's definitely better.

**Comment by C0urante** [comment]
Looks great! Merged.

File: README.md [hunk 5]
```diff
@@ -1,12 +1,16 @@
...
+[![Maven Central](https://img.shields.io/maven-central/v/com.wepay/kafka-connect-bigquery.svg?style=flat-square)](http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.wepay%22%20AND%20a%3A%22kafka-connect-bigquery%22)
 [![Open Issues](https://img.shields.io/github/issues/wepay/kafka-connect-bigquery.svg?style=flat-square)](https://github.com/wepay/kafka-connect-bigquery/issues)
 [![Pull Requests](https://img.shields.io/github/issues-pr/wepay/kafka-connect-bigquery.svg?style=flat-square)](https://github.com/wepay/kafka-connect-bigquery/pulls)
 [![Build Status](https://img.shields.io/travis/wepay/kafka-connect-bigquery.svg?style=flat-square)](https://travis-ci.org/wepay/kafka-connect-bigquery)
+[![Code Coverage](https://img.shields.io/codecov/c/github/wepay/kafka-connect-bigquery.svg?style=flat-square)](https://codecov.io/gh/wepay/kafka-connect-bigquery)
+[![Code Quality](https://img.shields.io/codacy/grade/547616ddaf5141bc9b10cd48fa9e0acb.svg?style=flat-square)](https://www.codacy.com/app/skyzyx/kafka-connect-bigquery/dashboard)
```
Comment (criccomini) [inline README.md]: This looks like it's using your personal @skyzyx github account. Can you switch it over to the WePay one?

Comment (skyzyx) [inline README.md]: We don't currently have one. I signed-up with my GitHub account, so I'll need to probably create a new account all-together for the team. That's a better idea, but I just need to do it first.

**Pull request merged**